metricbeat/module/windows/perfmon/pdh_windows.go

```diff
@@ -77,6 +80,26 @@ func PdhGetFormattedCounterValue(counter PdhCounterHandle, format PdhCounterForm
 	return counterType, &value, nil
 }
 
+func PdhGetFormattedCounterArray(counter PdhCounterHandle, format PdhCounterFormat) (int, *[]PdhCounterValueItem, error) {
```

Comment:
The returned slice is larger. So we need the `bufferCount` value to iterate over the slice to get the items which has a value.